doc/controls/ShadowContainer.md

```diff
@@ -49,8 +49,25 @@ Opacity | double | The opacity of the shadow.
 BlurRadius | double | The radius of the blur that will be applied to the shadow **[0..100]**.
 Spread | double | The spread will inflate or deflate (if negative) the control shadow size **before** applying the blur.
 
+As mentioned `IsInner` is similar to `inset` of `box-shadow` in CSS. See below an image that illustrate its usage:
+
+![CSS box-shadow without inset](../assets/box-shadow.png)
+
+Without `inset` a Drop (outer) Shadow is rendered.
+
+![CSS box-shadow with inset](../assets/box-shadow-inset.png)
+
+With `inset` an Inner Shadow is rendered.
+
+For more info of how `inset` works in CSS for `box-shadow` follow [this link](https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow#values).
+
+> [!IMPORTANT]
+> Be careful not to confuse the `Shadow` property with the `Shadows` property. The singular `Shadow` property comes from `UIElement`. To add shadows to the `ShadowContainer`, use only the `Shadows` property.
+
 ## Usage
 
+Observe shadows usage with the following code snippets and its results below.
```

Comment:
General documentation guidance is to write naturally as if speaking to a real person. This does not sound like anyone talks. - apologies if you do talk like this.